File: src/main/java/net/tastypommeslul/darkDupe/DarkDupe.java
package net.tastypommeslul.darkDupe;

import net.tastypommeslul.darkDupe.commands.DupeCommand;
import org.bukkit.plugin.java.JavaPlugin;

public class DarkDupe extends JavaPlugin {
    @Override
    public void onEnable() {
        getServer().getCommandMap().register("dupe", new DupeCommand());
    }
}

File: src/main/kotlin/net/tastypommeslul/darkDupe/DarkDupe.kt
package net.tastypommeslul.darkDupe

import net.tastypommeslul.darkDupe.commands.DupeCommand
import org.bukkit.plugin.java.JavaPlugin

class DarkDupe : JavaPlugin() {

  override fun onEnable() {
    server.commandMap.register("darkdupe", DupeCommand())
  }

  override fun onDisable() {
    // Plugin shutdown logic
  }
}
